boggle.js

```diff
@@ -21,9 +30,48 @@ function printBoggleBoard(boggleBoard) {
  */
 function shake(boggleBoard) {
   // This is your job. :)
+  let arrayOfDice = [
+    'AAEEGN',
+    'ELRTTY',
+    'AOOTTW',
+    'ABBJOO',
+    'EHRTVW',
+    'CIMOTU',
+    'DISTTY',
+    'EIOSST',
+    'DELRVY',
+    'ACHOPS',
+    'HIMNQU',
+    'EEINSU',
+    'EEGHNW',
+    'AFFKPS',
+    'HLNNRZ',
+    'DEILRX',
+  ];
+
+  let letterArray = [];
+  for (let i = 0; i < arrayOfDice.length; i++) {
+    for (let j = 0; j < 1; j++) {
```

Comment:
What is this inner loop here for? It runs the loop body exactly once every time.